packages/addons/service/multimedia/vdr-addon/source/bin/vdr.start

```diff
@@ -133,9 +133,24 @@ if [ "$ENABLE_XMLTV2VDR" == "true" ] ; then
     cp $source /var/lib/epgsources
   done
 fi
+
 if [ "$ENABLE_RESTFULAPI" == "true" ] ; then
-  VDR_ARG="$VDR_ARG -P 'restfulapi --ip $RESTFULAPI_IP --port $RESTFULAPI_PORT'"
+  VDR_ARG="$VDR_ARG -P 'restfulapi --ip $RESTFULAPI_IP --port $RESTFULAPI_PORT"
+  if [ ! -z $RESTFULAPI_EPGIMAGES ]; then
+    if [ ! -f $RESTFULAPI_EPGIMAGES ]; then
+      mkdir -p $RESTFULAPI_EPGIMAGES
+    fi
+    VDR_ARG="$VDR_ARG --epgimages=$RESTFULAPI_EPGIMAGES"
+  fi
+  if [ ! -z $RESTFULAPI_CHANNELLOGOS ]; then
+    if [ ! -f $RESTFULAPI_CHANNELLOGOS ]; then
+      mkdir -p $RESTFULAPI_CHANNELLOGOS
+    fi
+    VDR_ARG="$VDR_ARG --channellogos=$RESTFULAPI_CHANNELLOGOS"
```

Comment:
missing guard agains dirs with spaces/special chars. a user can select anything and vdr will fail to start. see reccmd above